Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -325,6 +332,23 @@ open class CarPlayMapViewController: UIViewController {
         navigationMapView.navigationCamera.follow()
     }
     
+    open override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        if #available(iOS 13.0, *) {
+            applyStyleIfNeeded(sessionConfiguration.contentStyle)
+        }
+    }
+    
+    @available(iOS 13.0, *)
+    func applyStyleIfNeeded(_ contentStyle: CPContentStyle) {
+        if contentStyle.contains(.dark) {
+            styleManager?.applyStyle(type: .night)
```

Comment:
Turns out that `StyleManager` changes on CarPlay will cause re-drawing on iOS as well. This is undesired effect, will be fixed in scope of: https://github.com/mapbox/mapbox-navigation-ios/issues/3883.